Avoid
    ccache: error: Failed to create directory /opt/local/var/macports/build/.ccache/tmp: Operation not permitted
--- mono/btls/CMakeLists.txt.orig	2019-01-17 00:45:20.000000000 -0700
+++ mono/btls/CMakeLists.txt	2019-02-17 15:18:40.000000000 -0700
@@ -12,11 +12,6 @@
 enable_language(C)
 enable_language(CXX)
 
-find_program(CCACHE_PROGRAM ccache)
-if(CCACHE_PROGRAM)
-    set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE "${CCACHE_PROGRAM}")
-endif()
-
 if (MSVC OR CYGWIN)
 	set(BTLS_HOST_WIN32 1)
 endif ()
